src/web/mjs/connectors/AnimeUnity.mjs

```diff
@@ -0,0 +1,89 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+export default class AnimeUnity extends Connector {
+    constructor() {
+        super();
+        super.id = 'animeunity';
+        super.label = 'AnimeUnity';
+        this.tags = [ 'anime', 'italian' ];
+        this.url = 'https://www.animeunity.tv';
+        this.token = undefined;
```

Comment:
If possible avoid fields that hold some kind of temporary state.
Consider to get the token as local variable inside of `_getMangas` and then provide it as parameter for the pagination e.g., `const mangas = this._getMangasFromPage (page, token)`